CHANGELOG.md

```diff
@@ -46,6 +46,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added: Research Site Open Hatches as available doors for Door Lock Rando.
 - Added: New option to place DNA anywhere.
 - Added: New option to force Save Station doors to be normal doors.
+- Added: If the user starts with random items, then an item collection screen will now be shown, telling the player which items they start with.
+- Added: Clearer GUI symbols, when expansions have been collected, but not their corresponding launcher.
+- Added: When softlock prevention is active, then the first two crumble blocks in Super Missile Chamber will be shoot blocks instead.
+- Changed: When softlock prevention is active, then in the EMP Escape route room, instead of the bottom row of speedbooster blocks being gone, now every pillar but the leftmost one is gone.
```

Comment:
Oh boy, don't make me correct the english grammar😥
`if` instead of `when` because in both cases it is really conditional?

The last one is very spaghetti IMHO.
`If softlock prevention is active, in the EMP Escape route room the bottom row of speedbooster blocks is no longer removed, but every pillar except the leftmost one` ???

Reply:
tried to rephrase it a little

Reply:
I still think it needs to be `If softlock prevention [...]` in 51.
In the rephrased one, the `is` at the end sounds off to me.
But I don't want to correct grammar because I'm just bad at my non native language. So, I take the wild card: @dyceron please 🙂